index.html

```diff
@@ -1,54 +1,162 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script async src="https://www.googletagmanager.com/gtag/js?id=G-CRE1ZD588G"></script>
+  <script>
+    window.dataLayer = window.dataLayer || [];
+    function gtag() { window.dataLayer.push(arguments); }
+    gtag('js', new Date());
+    gtag('config', 'G-CRE1ZD588G');
+  </script>
   <link rel="preconnect" href="https://fonts.googleapis.com">
-<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-<link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,400;0,600;0,800;1,500&display=swap" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,400;0,600;0,800;1,500&display=swap"
+    rel="stylesheet">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
   <link rel="stylesheet" href="assets/css/styles.css">
-  <title>Welcome: I'm a Software Developer!</title>
+  <title>Emmy - Full Stack Web Developer</title>
 </head>
+
 <body>
   <header>
-    <nav>
-      <div class="nav-container">
-        <div class="logo">
-          <a href="#"><img src="assets/images/mobile/My-Logo.svg" alt="Logo"></a>
-        </div>
-        <div class="menu-icon">
-          <a href="#"><img src="assets/images/mobile/Icon-Menu.svg" alt="Menue"></a>
-        </div>
-      </div>
+    <div class="logo">
+      <a href="#"><img src="assets/images/mobile/My-Logo.svg" alt="Portfolio-Logo"></a>
+    </div>
+    <div class="menu-icon" onclick="openNav()" id="click-image">
+      <a href="#"><img src="assets/images/mobile/Icon-Menu.svg" alt="Navbar-Menue"></a>
+    </div>
+    <nav id="myNav" class="overlay">
```

Comment:
- Kindly work on your design... Your mobile menu are appearing outside it:
![Screen Shot 2021-10-14 at 15 05 12](https://user-images.githubusercontent.com/22258274/137387893-f2ce6edf-4a20-46e0-af47-71243070234f.png)